app/views/pages/articles/2016-11-17-ruby-au-elections.html.md

```diff
@@ -0,0 +1,48 @@
+# Ruby Au Elections
+
+
+We have 5 positions open, President and Secretary, as well as two full-term and one half-term general position. Below is a brief description of each position, and a link to a nomination form to help streamline the meeting. If you are nominating someone other than yourself, please be sure to check with them first.
+
+Each committee member is expected to attend a monthly 1 hour online meeting (scheduled around all members other commitments), and to have regular access to Slack, 1password and Google docs. These are minimal requirements for staying in contact with the rest of the committee and accessing required files and login information. The time outputs outlined below are loose, with more time often needed at crucial points throughout the year and less at other times. Critical points are usually: turnover of committee members (after each Rails Camp), the weeks before RubyConf and Rails Camps and the turnover of the financial year.
+
+
+[nominations](https://docs.google.com/forms/d/e/1FAIpQLSdK7a3N3-5XT5LRqZIVBqEW1NgcxNTZJqaVs7kHwgwyZ1wDog/viewform)
```

Comment:
This link should be made more prominent. Something like "Nominate for a committee position here". Perhaps also move it up to the first paragraph.